.eslintrc

```
@@ -104,6 +104,7 @@
     "react/forbid-component-props": ["error", {
       "forbid": ["for"]
     }],
+    "react/no-array-index-key": "error",
```

```suggestion
    "react/no-array-index-key": "warn",
```

Let's make it a warning for now so we can already merge this and get the other pods to fix the issues.

Once that's done we can turn it into an error again.

---

Agreed!